x-pack/legacy/plugins/siem/public/components/formatted_date/index.tsx

```diff
@@ -118,7 +125,13 @@ export const FormattedRelativePreferenceDate = ({ value }: { value?: string | nu
       {moment(date)
         .add(1, 'hours')
         .isBefore(new Date()) ? (
-        <PreferenceFormattedDate data-test-subj="preference-time" value={date} />
+        <>
+          {labelOn &&
```

Comment:
I think we should avoid adding labels like `on` and `at` in this component. I feel like it can get really big really fast if we follow this pattern.  If you agree, we should create a new component and wrap this one.

Reply:
removed, added a note and a to do